workers/api-server/Cargo.toml

```diff
@@ -3,6 +3,15 @@ name = "api-server"
 version = "0.1.0"
 edition = "2021"
 
+[[test]]
+name = "integration"
+path = "integration/main.rs"
+harness = false
+required-features = ["test_helpers"]
+
+[features]
+test_helpers = ["darkpool-client/arbitrum"]
```

Comment:
Why not let `arbitrum` or `base` feature flags be set at the time of running the tests?